Docs: Improve CSP suggestion (#5875)
ILIAS-eLearning/ILIAS: docs/configuration/secure.md

see also: [Browser compatibility of HTTP headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP#Browser_compatibility)

> The proposed CSP is to be understood as a starting point. ILIAS as a generic software to support LMS scenarios could

Reviewer: ```suggestion
The proposed CSP is to be understood as a starting point. ILIAS is a generic software to support many different LMS scenarios and thus can't
```

> not provide a suggestion which fits all circumstances by guaranteeing the best security.

Reviewer: ```suggestion
provide a suggestion which fits all circumstances and guarantees the best security.
```

> Depending on your (especially external) user content (liked embedded media, SCORM packages, etc.) you should try to

Reviewer: ```suggestion
Depending on the content your users can provide (embedded media, SCORM packages, etc.) you should try to
```

initially define a CSP which is as strict as possible.

You could use the [Reporting Feature](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP#enabling_reporting) to

> log and analyse CSP violations to finally make the CSP (carefully) more loose, if necessary.

Reviewer: ```suggestion
log and analyse CSP violations to loosen the CSP gradually and carefully, if necessary.
```
